app/api/views/briefs.py

```diff
@@ -140,7 +140,11 @@ def get_brief_responses(brief_id):
     if not brief:
         not_found("Invalid brief id '{}'".format(brief_id))
 
-    brief_responses = brief_responses_service.get_brief_responses(brief_id, current_user.supplier_code)
+    supplier_code = getattr(current_user, 'supplier_code', None)
+    if supplier_code:
+        brief_responses = brief_responses_service.get_brief_responses(brief_id, supplier_code)
+    else:
+        brief_responses = []
```

Comment:
it might be an idea here to return all brief responses for a brief when the user is a buyer and have access to the brief.  It has happened a couple of times this information is asked for and can be used for the csv generation or brief dashboard when the time comes